src/components/tldraw/tldraw_cursor.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { useCurrentIndex } from 'components/utils/hooks';
+import storage from 'utils/data/storage';
+import './index.scss';
+
+const getCursor = (index, size, pageState) => {
+  const inactive = {
+    x: -1,
+    y: -1,
+  }
+
+  if (index === -1) return inactive;
+
+  const currentData = storage.cursor[index];
+  if (currentData.x === -1 && currentData.y === -1) return inactive;
+
+  let _x = null;
+  let _y = null;
+
+  _x = (currentData.x + pageState?.camera?.point[0]) * pageState?.camera?.zoom;
+  _y = (currentData.y + pageState?.camera?.point[1]) * pageState?.camera?.zoom;
+
+  if (_x > size.width || _y > size.height ) return inactive;
+
+  return {
+    x: _x,
+    y: _y
+  };
+};
+
+const Cursor = ({ tldrawAPI, size }) => {
```

Comment:
I'm not a big fan of using "snake case" for naming JS files. Since this component is already inside a tldraw components' folder and also named `Cursor` it could be just cursor.js